src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -5,35 +5,32 @@
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.hyperwallet.clientsdk.HyperwalletException;
 import com.hyperwallet.clientsdk.model.HyperwalletErrorList;
+import com.hyperwallet.clientsdk.model.HyperwalletVerificationDocument;
 import com.nimbusds.jose.JOSEException;
-import com.sun.jersey.api.client.Client;
-import com.sun.jersey.api.client.ClientResponse;
-import com.sun.jersey.api.client.WebResource;
-import com.sun.jersey.api.client.config.ClientConfig;
-import com.sun.jersey.api.client.config.DefaultClientConfig;
-import com.sun.jersey.api.client.filter.HTTPBasicAuthFilter;
-import com.sun.jersey.multipart.FormDataMultiPart;
-import com.sun.jersey.multipart.impl.MultiPartWriter;
-
-import javax.ws.rs.core.MediaType;
+
 import javax.xml.bind.DatatypeConverter;
-import java.io.IOException;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
 import java.text.ParseException;
 import java.util.HashMap;
+import java.util.Map;
+import java.nio.file.Files;
+import java.io.File;
+import java.util.List;
 
 public class HyperwalletApiClient {
 
     private static final String CONTENT_TYPE_HEADER = "Content-Type";
     private static final String VALID_JSON_CONTENT_TYPE = "application/json";
     private static final String VALID_JSON_JOSE_CONTENT_TYPE = "application/jose+json";
-
+    private HttpURLConnection httpConn;
     private final String username;
     private final String password;
     private final String version;
     private final HyperwalletEncryption hyperwalletEncryption;
     private final boolean isEncrypted;
-    private WebResource webResource;
-    private Client client;
+    private final String boundary = "--001103040245431341";
```

Comment:
Is this boundary different from the one you have on MultiPartRequest? Should it be a static constant and use the same everywhere?